clients/vulnerable-code/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Commit message: Why "service" and not "client"?

Reply:
What's the purpose of adding this project stub without actually adding any code to the project? I'd prefer to get this squash with the next commit.

Reply:
Squashed the commit.